README.md

```
@@ -58,37 +63,47 @@ cmake --build build --config Release
 docker build -t prevail .
 docker run -it prevail ebpf-samples/cilium/bpf_lxc.o 2/1
 1,0.009812,4132
 # To run the Linux verifier you'll need a privileged container:
 docker run --privileged -it prevail ebpf-samples/linux/cpustat_kern.o --domain=linux
 ```
+
 </details>
 
 ### Example:
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)